### DISCUSSION TITLE
How do I query the publication status of a submitted notice (API)

### DISCUSSION THREAD

**Discussion by JasperBeaurain** [OP-TED/eForms-SDK#242] [Q&A] [answered]
Hi,

As the subject says, how can I see the publication status of a notice I submitted?
The docs/openapi don't seem to describe an endpoint for this? Unless this is included in the validation-report endpoint somehow?

I know you are still working on this stuff, but the documentation is very incomplete at the moment...

Best regards,
Jasper

**Comment by pcolast** [comment, marked as answer]
Hello Jasper,

You can use searchProceduresEsentool
GET /api/v2/notice/notices/search-esentool

This will provide the list of notices with their details, including the notice publication status.

Regards,
Stefan
